Library/Homebrew/tap.rb

```diff
@@ -3,11 +3,19 @@ class Tap
 
   extend Enumerable
 
+  # The user name of this {Tap}.
```

Comment:
I guess it's more a comment on inline API documentation in general. I think the variable names already explain most of these comments e.g. `user` in a `Tap` class is obviously the user name of the Tap but it's not obvious that this is a user on GitHub or used as the first part of `user/repo` or has to be populated even for a non-github user. Does that make sense? Those are the sorts of questions it's worth answering on API docs like this, I think. It takes a little longer but it means someone can jump into the `Tap` class without context and understand a lot more about what and why we're doing. Thanks for your work here, though, I hope that's not too rude to suggest this.